images/node/example/Dockerfile

```diff
@@ -1,16 +1,19 @@
 # syntax=docker/dockerfile:1
 # Based on https://docs.docker.com/language/nodejs/build-images/
 
-ARG BASE=cgr.dev/chainguard/node
-
-FROM ${BASE}
+ARG BUILDER=cgr.dev/chainguard/node:latest-dev
```

Comment:
Should we be passing this ARG in any test modules?

Reply:
Not currently, but this will become useful for testing future changes.